src/cli/subcommands/sync_cmd.rs

```diff
@@ -176,23 +191,36 @@ fn format_tipset_cids(cids: &str) -> &str {
     }
 }
 
-/// Check if the snapshot download is in progress, if it is then wait till the snapshot download is done
-async fn check_snapshot_progress(client: &rpc::Client) -> anyhow::Result<()> {
+/// Check if the snapshot download is in progress, if wait is true,
+/// wait till snapshot download is completed else return after checking once
+async fn check_snapshot_progress(
+    client: &rpc::Client,
+    wait: bool,
+) -> anyhow::Result<SnapshotProgressState> {
     let mut interval = time::interval(Duration::from_secs(5));
     let mut stdout = stdout();
     loop {
         interval.tick().await;
         let progress_state = client.call(SyncSnapshotProgress::request(())?).await?;
-        match progress_state {
+        match &progress_state {
             SnapshotProgressState::InProgress { message } => {
-                println!("🌳 Snapshot download in progress: {}", message);
-                write!(
-                    stdout,
-                    "\r{}{}",
-                    anes::ClearLine::All,
-                    anes::MoveCursorUp(1)
-                )?;
-                continue;
+                // if wait is true, wait till snapshot download is completed
+                match wait {
+                    false => {
+                        println!("🌳 Snapshot download in progress");
```

Comment:
```suggestion
                        println!("🌳 Snapshot download in progress: {message}");
```

Reply:
@LesnyRumcajs 

I was planning with the same but the output will look little weird, since we are going print it only once:

If the snapshot progress just started we will get:
 - 🌳 Snapshot download in progress: Loading progress...
 
 If the snapshot is progress (downloading started) we will get:
 
  - 🌳 Snapshot download in progress:  1 GiB / 1 GiB, 100%, 512 MiB/s, elapsed time: 1s
  
Is this output acceptable?

Reply:
Done.